includes/admin/class-wcmp-export.php

```diff
@@ -1427,6 +1429,15 @@ private function saveOrderCollection(array $orderIds): array
             $dropOffPoint           = $configuration ? $configuration->getDefaultDropOffPoint() : null;
             $shipmentOptions        = $deliveryOptions->getShipmentOptions();
 
+            if ($dropOffPoint && CarrierInstabox::NAME === $carrier->getName()) {
```

Comment:
Dit zou m.i. uit de sdk moeten komen, een `Carrier::mustHaveDropOffPoint` o.i.d., zodat je niet specifieke carrier namen hoeft te gaan checken in de plugins. Daar komt bij, het lijkt of deze en fout gooit wanneer er een dropoffpoint aanwezig is? hij checkt ($dropoffpoint), niet (! $dropoffpoint). Als dit wel is hoe het hoort, is de code te ingewikkeld voor mij, misschien kan hij versimpeld worden?